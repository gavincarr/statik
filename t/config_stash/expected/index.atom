<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
  <title>My Bl√∏g</title>
  <subtitle></subtitle>
  <link href="/"/>
  <updated>2011-01-05T10:03:00+00:00</updated>
  <author>
    <name>Jane Author</name>
  </author>
  <id>tag:example.com,2011:/</id>
  <generator uri="https://github.com/gavincarr/statik" version="0.1">Statik</generator>
  <entry>
    <title>Post the fifth</title>
    <link href="/sub2/fifth"/>
    <id>tag:example.com,2011-01-05:/sub2/fifth</id>
    <updated>2011-01-05T10:03:00+00:00</updated>
  </entry>
  <entry>
    <title>Post the fourth</title>
    <link href="/fourth"/>
    <id>tag:example.com,2011-01-04:/fourth</id>
    <updated>2011-01-04T14:15:16+00:00</updated>
  </entry>
  <entry>
    <title>Post the third</title>
    <link href="/third"/>
    <id>tag:example.com,2011-01-03:/third</id>
    <updated>2011-01-03T20:20:20+00:00</updated>
  </entry>
  <entry>
    <title>Post the second</title>
    <link href="/sub1/second"/>
    <id>tag:example.com,2011-01-02:/sub1/second</id>
    <updated>2011-01-02T08:30:00+00:00</updated>
  </entry>
  <entry>
    <title>Post the first</title>
    <link href="/first"/>
    <id>tag:example.com,2011-01-01:/first</id>
    <updated>2011-01-01T12:00:00+00:00</updated>
  </entry>
</feed>
