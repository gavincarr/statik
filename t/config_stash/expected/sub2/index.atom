<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
  <title>My Bl√∏g</title>
  <subtitle></subtitle>
  <link href="/sub2/"/>
  <updated>2011-01-05T10:03:00+00:00</updated>
  <author>
    <name>Jane Author</name>
  </author>
  <id>tag:example.com,2011:/sub2/</id>
  <generator uri="https://github.com/gavincarr/statik" version="0.1">Statik</generator>
  <entry>
    <title>Post the fifth</title>
    <link href="/sub2/fifth"/>
    <id>tag:example.com,2011-01-05:/sub2/fifth</id>
    <updated>2011-01-05T10:03:00+00:00</updated>
  </entry>
</feed>
