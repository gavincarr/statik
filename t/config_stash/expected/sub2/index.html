<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>My Bløg</title>
<link rel="stylesheet" type="text/css" href="style-original.css" />
<link rel="alternate" type="application/atom+xml" title="My Bløg feed" href="/index.atom" />
</head>
<body>
<div id="header">
<h1>My Bløg</h1>
<p class="tagline"></p>
</div>
<div id="body">
<div id="article">
<h2><a href="/sub2/fifth" rel="bookmark" title="link to this post">Post the fifth</a></h2>
<div id="timestamp">2011-01-05</div>
<p>Nam justo augue, dictum a, hendrerit in, ultricies in, leo. Nullam
eleifend. Duis tempor ipsum vitae diam. Curabitur felis dui, bibendum
vitae, luctus quis, volutpat sed, orci. Cras vulputate ullamcorper ante.
Sed congue libero in orci. Vivamus a odio ac sapien dignissim posuere.
Sed mollis ipsum id libero. Quisque vitae justo. Nulla vitae mauris.
Phasellus convallis ligula in nulla. Morbi aliquet, velit ac semper
iaculis, mi odio dignissim diam, id dapibus eros metus id nisi. Nulla
vitae sapien. Nulla ligula. Cum sociis natoque penatibus et magnis dis
parturient montes, nascetur ridiculus mus. Curabitur non nisl id ante
egestas dapibus. Sed mollis ornare ipsum. In id enim dignissim erat
viverra vulputate.</p>
</div>
</div>
<div id="footer">
Copyright 2011 Jane Author.
</div>
</body>
</html>
